bitnami/contour-operator/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if and .Values.metrics.serviceMonitor.enabled .Values.metrics.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "common.names.fullname" . }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    {{- if .Values.metrics.serviceMonitor.selector }}
```

Comment:
I've seen this in other charts and I think it is a mistake. If the value is called selector then it should go to the `selector` section. Otherwise, we would have a value called selector that selects nothing. Check the recent `geode` chart to see how to use it.